.github/workflows/performance.yml

```diff
@@ -77,6 +77,14 @@ jobs:
                   WP_MAJOR="${WP_VERSION_ARRAY[0]}.${WP_VERSION_ARRAY[1]}"
                   ./bin/plugin/cli.js perf $GITHUB_SHA debd225d007f4e441ceec80fbd6fa96653f94737 --tests-branch $GITHUB_SHA --wp-version "$WP_MAJOR"
 
+            - name: Archive debug artifacts (screenshots, traces)
+              uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+              if: always()
+              with:
+                  name: failures-artifacts
+                  path: ./__test-results/artifacts
+                  if-no-files-found: ignore
```

Comment:
if looking to do this for PRs we may want to move this up to line 53 where it's associated with those, and we may not want to upload for release or trunk-merge events.

Reply:
Moved in https://github.com/WordPress/gutenberg/pull/48243/commits/39d517f40ccda52e13ce09ff4d7f79b37fd610a2.

Reply:
I moved the action back to the bottom because every `Compare...` action can produce artifacts since they all call the `./bin/plugin/cli.js perf` script. I think it should be running `always()` as it should always attempt to upload the artifacts, regardless of the failed action.